bin/pyenv-installer

```diff
@@ -89,4 +89,11 @@ if ! command -v pyenv 1>/dev/null; then
       ;;
     esac
   } >&2
+elif [ -d "${PYENV_ROOT}" ]; then
+  { echo
+    colorize 1 "WARNING"
+    echo ": Kindly remove '.pyenv' from ${HOME} to proceed with installation."
```

Comment:
I like the polite way the computer speaks to me here, but maybe we should be a bit clearer? Something like:

": cannot proceed with installation - kindly remove '.pyenv' from ${HOME} first."